lib/wellies_web/temperature/current.ex

```diff
@@ -0,0 +1,31 @@
+defmodule ForecastNow do
+  defstruct [:date, :hour, :temperature]
+end
+
+defmodule WelliesWeb.CurrentTemperature do
+  alias WelliesWeb.ResponseHandler
+  alias WelliesWeb.OpenWeatherParser
+  alias WelliesWeb.OpenWeatherApi
+
+  def request_current_forecast(id) do
+    id
+    |> OpenWeatherApi.hourly_in
```

Comment:
I would call this just `hourly`, since the `in` doesn't really provide much value, but maybe add confusion. (in something?)